pkgs/by-name/ni/nix-required-mounts/package.nix

```diff
@@ -0,0 +1,47 @@
+{ addOpenGLRunpath
+, cmake
+, allowedPatterns ? rec {
+    opengl.onFeatures = [ "opengl" ];
+    opengl.paths = [
+      addOpenGLRunpath.driverLink
+      "/dev/video*"
+      "/dev/dri"
+    ];
+    cuda.onFeatures = [ "cuda" ];
+    cuda.paths = opengl.paths ++ [
+      "/dev/nvidia*"
+    ];
+  }
```

Comment:
Indeed. The package and the module end up being different because the module also includes the symlink targets in the nix store, whereas the package in principle could be used on a fhs machine.

But maybe I just put the shared part into the passthru